### PR TITLE
more info printed to logger in plot_simtel_histograms

### LINKED ISSUE #661: simtools-plot-simtel-histograms info messages not complete
When running `simtools-plot-simtel-histograms` we get only two print out messages:
```
INFO::base(l60)::__init__::Opening new file simtel-data/North/gamma/data/run000001_gamma_za020deg_azm000deg_North_TestLayout_test-production.simtel.zst
INFO::base(l93)::__init__::Found zstd compressed file
```
I propose we increase the amount of information, especially if the script was ran successfully.

### PR DESCRIPTION
**Pull request by VictorBarbosaMartins**
Closes #661.
Small pull request to solve #661. It should add some info messages to the logger of `plot_simtel_histograms`.

You can test it by running e.g. ```simtools-plot-simtel-histograms --hist_file_names tests/resources/run201_proton_za20deg_azm0deg_North_TestLayout_test-prod.simtel.zst --figure_name test_hist```

### CODE REVIEW

File: simtools/applications/plot_simtel_histograms.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def main():
     logger = logging.getLogger()
     logger.setLevel(gen.get_log_level_from_user(args_dict["log_level"]))
     n_lists = len(args_dict["hist_file_names"])
```
Comment (GernotMaier) [inline simtools/applications/plot_simtel_histograms.py:70]: Suggest to condense this into one sentence: `Opening {n_lists} histogram files` (don’t call it hist_file_names, this is hard to understand)

File: simtools/applications/plot_simtel_histograms.py [hunk 2]
```diff
@@ -86,14 +86,15 @@ def main():
             raise TypeError
 
     # Building SimtelHistograms
+    logger.info(f"Histograms will be produced for {histogram_files}.")
```
Comment (GernotMaier) [inline simtools/applications/plot_simtel_histograms.py:89]: Tiny comment, but maybe we have consistent language. My suggestion is to use the same as in line 97: `Creating pdf file`; `Creating histograms for {histogram_files}`

(Avoiding philosophical discussions about the differences between `creating` and `producing` :-) )

**Review by GernotMaier — APPROVED** [review]
Tiny comments, otherwise good!

File: simtools/applications/plot_simtel_histograms.py (hunk 1 above)
Comment (VictorBarbosaMartins) [reply simtools/applications/plot_simtel_histograms.py:70]: Good point. I updated it to `Opening {n_lists} files` as the files can also be text files with lists of histogram files.

**Pull request merged**